src/input.cpp

```diff
@@ -139,25 +139,25 @@ void input_manager::init()
     }
     t_actions &actions = action_contexts["DEFAULTMODE"];
     std::set<action_id> touched;
-    for( std::map<char, action_id>::const_iterator a = keymap.begin(); a != keymap.end(); ++a ) {
-        const std::string action_id = action_ident( a->second );
+    for( const auto &a : keymap ) {
```

Comment:
```suggestion
    for( const std::pair<char, action_id> &a : keymap ) {
```

Reply:
Again, clang 10 probably wants
```suggestion
    for( const std::pair<const char, action_id> &a : keymap ) {
```